autoscale/src/main/java/com/sequenceiq/periscope/monitor/UpdateMonitor.java

```diff
@@ -0,0 +1,50 @@
+package com.sequenceiq.periscope.monitor;
+
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
+
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.atomic.AtomicLong;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.periscope.model.ScalingActivities;
+import com.sequenceiq.periscope.monitor.evaluator.EvaluatorExecutor;
+import com.sequenceiq.periscope.monitor.evaluator.update.ActivityUpdateEvaluator;
+import com.sequenceiq.periscope.service.ha.PeriscopeNodeService;
+
+@Component
+@ConditionalOnProperty(prefix = "periscope.enabledAutoscaleMonitors.update-monitor", name = "enabled", havingValue = "true")
+public class UpdateMonitor extends ScalingActivityMonitor {
+
+    // TODO: ID conflicts with CleanupMonitor?
+    private final AtomicLong scalingActivitiesIdGenerator = new AtomicLong(0L);
+
+    @Override
+    protected List<ScalingActivities> getMonitored() {
+        PeriscopeNodeService periscopeNodeService = getPeriscopeNodeService();
+        if (periscopeNodeService.isLeader(getPeriscopeNodeConfig().getId())) {
+            Set<Long> scalingActivityIds = getScalingActivityService().findAllIdsOfSuccessfulAndInProgressScalingActivity();
+            return scalingActivityIds.isEmpty() ? emptyList() : singletonList(new ScalingActivities(scalingActivitiesIdGenerator.incrementAndGet(),
+                    scalingActivityIds));
+        }
+        return emptyList();
+    }
+
+    @Override
+    public String getIdentifier() {
+        return "scaling-activity-update-monitor";
+    }
+
+    @Override
+    public String getTriggerExpression() {
+        return MonitorUpdateRate.EVERY_TEN_MIN_RATE_CRON;
```

Comment:
Can we run this more frequently. EVERY_2_MINUTES maybe? Using the CLI as a monitoring mechanism would need this to be updated often enough.

Reply:
Updated to run every 2 mins